server/routes/nuxt-api/__sitemap__/datasets.ts

```diff
@@ -0,0 +1,40 @@
+import { defineSitemapEventHandler } from '#imports'
+import type { SitemapUrl } from '#sitemap/types'
+
+export default defineSitemapEventHandler(async (event) => {
+  const config = useRuntimeConfig()
+
+  let { page = 1 } = getQuery(event)
```

Comment:
Maybe name it `sitemapPart` or something like this to avoid confusing with the API page since we have two different pagination?